lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -109,8 +109,8 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
         this.removeFile(file);
         if (file['id'] === this.selectedNode?.id) {
             this.selectedNode = null;
-            this.contentModelFormFileHandler();
         }
+        this.contentModelFormFileHandler(this.field.value.length > 0 ? this.field.value[0] : null);
     }
```

Comment:
The fix:
Regardless of whether the removed file was selected or not,
    if there are still files to display,
       display the first one.
    Otherwise,
        call contentModelFormFileHandler with null to stop displaying anything.